### ISSUE TITLE
Improvement to Portal 1 Style Emancipation Grills

### ISSUE THREAD

**Issue by ghost** [BEEmod/BEE2-items#1963]
Have the Custom Emancipation Grills;
![20170514094225_1](https://cloud.githubusercontent.com/assets/25728252/26034408/19194712-388a-11e7-8c6b-50adbe28cfb3.jpg)
Have a wispy-sorta look, like this:
![20170514094228_1](https://cloud.githubusercontent.com/assets/25728252/26034411/27fc2cb8-388a-11e7-9dd4-0357e0b0534a.jpg)
Just have the wisps match the color of the field.

**Comment by TeamSpen210** [comment]
Unfortunately we can't pack the particles needed for that.

**Comment by hop193** [comment]
:(

**Comment by Luke18033** [comment]
We can now pack particles, not sure if this would be a good idea though.

**Comment by TeamSpen210** [comment]
Ooh yeah, might be neat.

**Comment by TeamSpen210** [comment]
It does need to be more clear as to which is which though.

**Comment by Luke18033** [comment]
Yeah. Different colored particles would mostly work, though they could be difficult to distiguish especially for people who are colorblind. Maybe the particles for each field should have unique movement patterns? And fields like the PRF which block things should use a texture closer to P2 fizzlers, to give them a more solid look.

**Comment by Luke18033** [comment]
Absolute Fizzlers should probably use the normal particles but thicker, faster-moving, and darker blue. Matter Inquisition Fields would maybe have the particles move in a swirl pattern?

Solid fizzlers should use a design based off the Still Alive blockfield, but recolored and with different patterns depending on the field (e.g. the grating pattern for PRFs and FDFs).
